modules/querying-bootstrap-node-journal-logs.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="querying-bootstrap-node-journal-logs_{context}"]
+= Querying bootstrap node journal logs
+
+Gather `bootkube.service` `journald` unit logs on a bootstrap node.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Your API service is still functional.
+* The {product-title} CLI (`oc`) installed.
+
+.Procedure
+
+* Query `bootkube.service` `journald` unit logs from a bootstrap node during {product-title} {product-version} installation:
+.. Determine the IP address of the bootstrap node:
++
+----
+$ python -c 'import sys, json; d=reduce(lambda x,y: dict(x.items() + y.items()), map(lambda x: x["resources"], json.load(sys.stdin)["modules"])); k="aws_instance.bootstrap"; print d[k]["primary"]["attributes"]["public_ip"] if k in d else ""' < cluster_install_dir/terraform.tfstate
```

Comment:
This is overly complex. They key piece of this is making sure its possible to identify the ip address of the boot strap system.

Reply:
Thanks Eric. On review, I think we just need a simple step here indicating that the reader needs to obtain the IP address for the bootstrap node, without going in to detail about how to get that:

~~~
.. Determine the IP address of the bootstrap node.
~~~